src/algoliasearch.helper.js

```diff
@@ -346,12 +347,12 @@ AlgoliaSearchHelper.prototype.isRefined = function( facet, value ) {
 };
 
 /**
- * Check if the facet has any disjunctive or conjunctive refinements
- * @param {string} facet the facet attribute name
- * @return {boolean} true if the facet is facetted by at least one value
+ * Check if the attribute has any numeric, disjunctive or conjunctive refinements
+ * @param {string} attribute the name of the attribute
+ * @return {boolean} true if the attribute is filtered by at least one value
  */
-AlgoliaSearchHelper.prototype.hasRefinements = function( facet ) {
-  return this.isRefined( facet );
+AlgoliaSearchHelper.prototype.hasRefinements = function( attribute ) {
+  return !isEmpty( this.state.getNumericRefinements( attribute ) ) || this.isRefined( attribute );
```

Comment:
Well for the two comments : 
 - we can't provide an API that fits the structure of both the facets (attribute, value) and the numerics (attribute, operator, value), the same way we can't provide that for tags (value). So we need those two methods, one that is specific to facets (isRefined) and one that is specific to attribute based refinements (hasRefinements)
 - Ok I will clean that up so it's easier to get what's going when reading the code.